decentraland/components/presentational/EthereumSignUpForm.js

```diff
@@ -44,6 +39,27 @@ export default class EthereumSignUpForm extends React.Component {
   }
 }
 
+export function TextInputField({ handleUsername, username }) {
```

Comment:
And maybe `onChange` instead of `handleUsername`?

Reply:
Made this change too